docs/manual/transform_data.rst

```diff
@@ -131,6 +131,118 @@ Transformation of a column based on a condition is possible via :func:`ifelse()`
     3 | even
    [4 rows x 1 column]
 
+Transforming a column via a boolean condition in the `i` section is possible; note that this could result in a reduced number of rows::
+
+   >>> DT[f.dates < dt.time.ymd(2020,1,1), f.integers ** 4]
+      |      C0
+      | float64
+   -- + -------
+    0 |       1
+    1 |      16
+   [2 rows x 1 column]
+
+
+Column type can be changed with the :func:`as_type()` function
+
+   - Single column::
+
+      >>> DT[:, dt.as_type(f.integers, str)]
+         | integers
+         | str32
+      -- + --------
+       0 | 1
+       1 | 2
+       2 | 3
+       3 | 4
+      [4 rows x 1 column]
+
+- Multiple columns::
+
+      >>> DT[:, dt.as_type(f['integers', 'floats'], str)]
+         | integers  floats
+         | str32     str32
+      -- + --------  ------
+       0 | 1         10.0
+       1 | 2         11.5
+       2 | 3         12.3
+       3 | 4         -13.0
+      [4 rows x 2 columns]
+
+Values across multiple columns can be replaced with the :meth:`datatable.Frame.replace` method; this works on the entire frame, and is an in-place operation.
+
+The :meth:`datatable.Frame.replace` syntax is -> ``frame.replace(replace_what, replace_with)``, where `replace_what` values in the frame are replaced with `replaced_with`::
+
+   >>> DF = DT.copy()
```

Comment:
Do we really need `DF` here and can't continue using `DT`? You change the types above anyways, so may be just call the original data `DT0` and make an assignment `DT = DT0` anytime you need to reset all the changes.

Reply:
for the dtype change and other column operation, there is no assignment; there is no change to the dataframe. `replace` changes the dataframe, which is why I create a copy of the dataframe before using it. without the `copy`, any change made to `DF` reflects in `DT`

Reply:
Yes, I understand. Just confused the type conversion with something else you did above, when you had to do `DT = DT[:, :4]`. 

Anyways, It seems to be a pretty standard use-case 
- you have the original data;
- adjust something;
- want to have the original data back for the purpose of a new section. 

I guess it is a good idea to keep using `DT` everywhere in the tutorial, so that people do not confuse different frames. Also, may be call the original data something like `DT0` or `DT_original` and anytime you want to reset your changes simply do `DT = DT0.copy()` or `DT = DT_original.copy()`. It could be more readable and standard than doing `DT = DT[:, :4]` or anything else you need  to do in a particular case.